DigitalOcean-public.v2.yaml

```diff
@@ -13,6 +13,11 @@ info:
 servers:
   - url: 'https://api.digitalocean.com/v2'
     description: production
+tags:
+  - name: account
+    description: Account
```

Comment:
We should either leave the description out or provide more than a one-word description that is the same as the tag....

Reply:
ah, plugging it into Swagger Hub I better see how these are getting used. 

![image](https://user-images.githubusercontent.com/46943/85446323-52469000-b562-11ea-86bb-61f3d3591bc4.png)

Updated